friends/src/components/Friend.js

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import "./fr.css"
+
+class Friend extends React.Component{
+    constructor(props) {
+        super(props)
+        this.state = {
+            inputName:this.props.friend.name,
+            inputAge: this.props.friend.age,
+            inputEmail:this.props.friend.email
+        }
+    }
+
+    deleteFriend = (event) => {
```

Comment:
This should be in your FriendList so it can have the response be set to your state for data. Doing the window.reload does not make this a friendly application for the user and actually defeats the purpose of single page applications